source/components/home/countdown/countdown.js

```diff
@@ -1,40 +1,70 @@
 import React, { Component } from 'react'
 import ReactDOM from 'react-dom';
+import $ from 'jquery'
+import styles from './styles.scss'
 
-
-Number.prototype.pad = function(size) {
+Number.prototype.pad = function (size) {
     var s = String(this);
-    while (s.length < (size || 2)) {s = "0" + s;}
+    while (s.length < (size || 2)) { s = "0" + s; }
     return s;
 }
 
 class Countdown extends Component {
-    
+
     constructor() {
         super();
     }
 
-    componentWillMount(){
-      var countDownDate = new Date("Sep 17, 2018 00:00:00").getTime();
-        var x = setInterval(function() {
+    componentWillMount() {
```

Comment:
Is there any way to make the circles animated? On first glance the way it's designed makes it look like the circle indicated how much time is left until another second/minute/hour/day elapses